Week1/homework/app.js

```diff
@@ -1,11 +1,183 @@
 'use strict';
 
-{
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
+// ------------1.1-----
+const bookTitles = [
+  'pragmatic_programmer',
+  'clean_code',
+  'code_complete',
+  'design_patterns',
+  'refactoring',
+  'mythical_man_month',
+  'javascript_good_part',
+  'structure_and_interpretation_of_computer',
+  'c_programming_language',
+  'head_first_design',
+];
 
-  // Replace with your own code
-  console.log(bookTitles);
+// --------1.2-------
+// console.log(bookTitles);
+
+// --------1.3------
+function createUl(ul) {
+  for (let i = 0; i < bookTitles.length; i++) {
+    const listTitels = bookTitles[i];
+    const listItems = document.createElement('li');
+    listItems.innerText = listTitels;
+    ul.appendChild(listItems);
+  }
+}
+function main() {
+  const root = document.getElementById('root');
+  const ul = document.createElement('ul');
+  root.appendChild(ul);
+  createUl(ul);
+}
+window.addEventListener('load', main);
+
+// -------1.4-------------
+const bookDetails = {
+  pragmatic_programmer: {
+    title: 'pragmatic programmer',
+    language: 'english',
+    author: 'Andy Hunt',
+    // pragmatic_programmer: './img/pragmatic_programmer.jpg',
+  },
+  clean_code: {
+    title: 'clean code',
+    language: 'english',
+    author: 'Robert C',
+  },
+  code_complete: {
+    title: 'code complete',
+    language: 'english',
+    author: 'Steve McConnel',
+  },
+  design_patterns: {
+    title: 'Design patterns',
+    language: 'english',
+    author: 'Erich Gamma',
+  },
+  Refracting: {
+    title: 'Refracting',
```

Comment:
Small typo, the book is called Refactoring.

Reply:
I correct it